Add "Configure and link your App Clips"

rogerluan opened the pull request:
## "Configure and link your App Clips" session.

**Link:** https://developer.apple.com/videos/play/wwdc2020/10146/
**Presenters:** _Ada Chan, Luming Yin_
**Duration:** 22:56

## Two Questions:

1. Should we add the duration of every video on the top of the section? I find it useful, but it'd require everyone to do the same. Open to suggestions.
2. This is more of a hands-on video, with code samples. Should we include code samples in these summaries? For brevity reasons, I left them out. Lemme know what's the standard here (and this should probably be documented in the `CONTRIBUTING.md` file 👍 )

rogerluan committed:
Add "Configure and link your app clips"

rogerluan committed:
Capitalize "App Clips" occurrences.

rogerluan commented on README.md:
```diff
@@ -421,4 +421,52 @@ _Matt Ricketson, Taylor Kelly_
+
+   _Ada Chan, Luming Yin_
+
+   Duration: 22:56
```
Should we add this to every video? Thoughts?

Blackjacx replied:
I updated the contribution rules: https://github.com/Blackjacx/WWDC/blob/master/CONTRIBUTING.md#submitting-changes

rogerluan replied:
Sweet, that works for me! 💪

rogerluan committed:
Fix indentation.

rogerluan committed:
Add ToC entry.

rogerluan committed:
Fix indentation.

Blackjacx commented on README.md:
```diff
@@ -421,4 +422,52 @@ _Matt Ricketson, Taylor Kelly_
+
+_Ada Chan, Luming Yin_
+
+Duration: 22:56
```
Please without the duration, see the new https://github.com/Blackjacx/WWDC/blob/master/CONTRIBUTING.md

Blackjacx commented on README.md:
```diff
@@ -421,4 +422,52 @@ _Matt Ricketson, Taylor Kelly_
+
+## Configure and link your App Clips
+
+https://developer.apple.com/videos/play/wwdc2020/10146/
```
If possible please use the short url version, it is easier to parse later on for me :-) You can just copy & paste one of the others and change the session number.

Blackjacx reviewed (changes requested):
Thanks for your input, really cool addition 🙏

rogerluan committed:
Merge branch 'master' of github.com:Blackjacx/WWDC into app-clips

# Conflicts:
#	README.md

rogerluan commented:
Sorry for putting all the fixes in a single merge commit 🤦 but I addressed your comments, fixed some typos in the new latest and removed redundant trailing whitespaces at the end of the file. :)

Blackjacx reviewed (approved):
All good bro, you are my lifesaver! Thanks for all the corrections. This is really nice and keeps up the quality of the notes 👍

Blackjacx committed:
Merge branch 'master' into app-clips
